messages/en.js

```diff
@@ -118,14 +118,14 @@ Blockly.Msg.SWITCH_CASE_TOOLTIP = "When a specific value is matched.";
 Blockly.Msg.SWITCH_CASE_LIST = "in case of";
 Blockly.Msg.SWITCH_CASE_LIST_TOOLTIP = "List of all available cases";
 Blockly.Msg.SWITCH_DO = "do";
-Blockly.Msg.MISC_JAVASCRIPT= "Javascript statement %1;";
-Blockly.Msg.MISC_JAVASCRIPT_TOOLTIP = "Enter Javascript code when no blocks offer certain functionality.";
-Blockly.Msg.MISC_JSEXPRESS= "Javascript expression %1";
+Blockly.Msg.MISC_JAVASCRIPT = "JavaScript statement %1;";
```

Comment:
Is the semicolon inside the string wanted, or is a typo?

Reply:
We use always a semicolon at the end of our JavaScript expressions and statement, even in our translated texts it seems 😉
No you are right: those can be removed here.

Reply:
Hang on  - isn't it there to show that the user doesn't have to put one in?

Reply:
I left it as is for now.

Reply:
Ok fine for me!